src/plugins/vis_type_xy/public/editor/components/options/point_series/point_series.test.tsx

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { mountWithIntl } from '@kbn/test/jest';
+import { ReactWrapper } from 'enzyme';
+import { PointSeriesOptions } from './point_series';
+import { findTestSubject } from '@elastic/eui/lib/test';
+import { act } from 'react-dom/test-utils';
+import { ChartType } from '../../../../../common';
+import { ValidationVisOptionsProps } from '../../common';
+import { VisParams } from '../../../../types';
+import { getAggs, getVis, getStateParams } from './point_series.mocks';
+
+jest.mock('../../../../services', () => ({
+  getTrackUiMetric: jest.fn(() => null),
+  getPalettesService: jest.fn(() => {
+    return {
+      getPalettes: jest.fn(),
+    };
+  }),
+}));
+
+const pointSeriesProps = {
+  aggs: getAggs(),
+  hasHistogramAgg: false,
+  extraProps: {
+    showElasticChartsOptions: false,
+  },
+  isTabSelected: false,
+  setMultipleValidity: jest.fn(),
+  setTouched: jest.fn(),
+  setValue: jest.fn(),
+  setValidity: jest.fn(),
+  stateParams: getStateParams(ChartType.Histogram, false),
+  vis: getVis('date_histogram'),
+};
+
+describe('PoitSeries Editor', function () {
+  let props: ValidationVisOptionsProps<
+    VisParams,
+    {
+      showElasticChartsOptions: boolean;
+    }
+  >;
+  let component: ReactWrapper<
+    ValidationVisOptionsProps<
+      VisParams,
+      {
+        showElasticChartsOptions: boolean;
+      }
+    >
```

Comment:
nit: let's avoid of duplicating any code. It's ok to create a new type in test if some some reason we don't want to export it.  

```
type PointSeriesOptionsProps = ValidationVisOptionsProps<
  VisParams,
  {
    showElasticChartsOptions: boolean;
  }
>;
```
but I like the following option more: 
`type PointSeriesOptionsProps = Parameters<typeof PointSeriesOptions>[0];`